### ISSUE TITLE
Remove old db files

### ISSUE THREAD

**Issue by gabrielasd** [theochem/AtomDB#76]
We need to remove the old compiled dataset versions here to avoid confusion/errors (e.g. #70).
Updated DB versions are being stored in a separate repository, [AtomDBdata](https://github.com/theochem/AtomDBdata/commits/main/), and there is no reason to have duplicated information.

**Comment by gabrielasd** [comment]
Also in relation to this change (separating package from DB) I think we should consider removing the db folders in this repository from being tracked by git.
@msricher , @marco-2023 what do you think?

**Comment by marco-2023** [comment]
We could (and probably should) remove then also from the history using `git filter-repo` or `bfg` https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository

**Comment by gabrielasd** [comment]
The commit d75f9128b5db16523f3f0e4fe32acd988b795160 removed the Slater DB files from `datasets/slate/db`

**Comment by msricher** [comment]
I uploaded a copy of the AtomDB repo, with old db files removed from the git history: https://github.com/msricher/AtomDB_clean

I'll document what I did here. This makes use of the `git-filter-repo` package (at least that's the name on Arch). 

1. `git log --all --name-only --pretty=format:%H -- ^'*.msg' | grep -E '*.msg$'` lists all of the files matching `^*.msg`, by commit. There may be some duplicates due to multiple commits modifying the same files but that doesn't matter. I filtered out the lines containing the commit hashes with `grep`.
2. `for i in $(cat msg_files | xargs); do git filter-repo --invert-paths --path "$i"; done` removes each of the files matched above from the git history.

Before I force-push this over the main repo, would either of you verify that the result in my repo looks correct? @gabrielasd @marco-2023
